<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>odunc_alma_islemleri</class>
 <widget class="QDialog" name="odunc_alma_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_Odunc_Alinan_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Uye_No">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye No:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Uye_No">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Tum_Uyeler">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Odunc_Alinan_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Stok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Alma Tarihi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Tum_Kitaplar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Kitaplar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Odunc_Alma_Islemleri">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÖDÜNÇ ALMA İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Kitap_No">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Odunc_Al">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Al</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Tum_Uyeler">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Üyeler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Kitap_No">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap No:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Tum_Kitaplar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Odunc_Alma_Tarihi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>15</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>6</month>
     <day>10</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>14</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1999</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
